.github/workflows/snyk_pr.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: snyk_pr
+
+on: [pull_request_target]
```

Comment:
I don't think we need this second workflow at all. `pull_request_target` means the base branch, not the PR itself, so the workflow will get run on merge anyway.